sklearn/calibration.py

```diff
@@ -980,8 +980,8 @@ class CalibrationDisplay:
     y_prob : ndarray of shape (n_samples,)
         Probability estimates for the positive class, for each sample.
 
-    name : str, default=None
-        Name for labeling curve.
+    estimator_name : str, default=None
+        Name of estimator. If None, the estimator name is not shown.
```

Comment:
nitpick:
```suggestion
        Name of estimator. If `None`, the estimator name is not shown.
```

Reply:
To be pedantic there is no label shown in the plot only if `self.estimator` is `None` and in `plot` method `name` is also `None`...?